acceptance-tests/features/steps/inform_all_borrowers_signed.rb

```diff
@@ -0,0 +1,50 @@
+Given(/^borrower one of two navigates to sign the deed page$/) do
+  visit($BORROWER_FRONTEND_URL + '/deed/search')
+  @deed = HTTP.get($DEED_API_URL + '/deed/' + @deed_id.to_s)
+  @borrower_token =
+    JSON.parse(@deed.body)['deed']['operative-deed']['borrowers'][0]['token']
+  fill_in('borrower_token', with: @borrower_token)
+  click_button('Search')
+end
+
+When(/^I enter the borrowers name$/) do
+  fill_in('user-name', with: 'Peter Smith')
+  click_button('Sign the deed')
+end
+
+Then(/^a message is displayed 'The following people still need to sign the mortgage deed'$/) do
+  expect(page).to have_content('The following people still need to sign the '\
+                               'mortgage deed')
```

Comment:
Have amended to do as suggested, also applied same logic to borrower2 being displayed.